dkist/net/attrs.py

```diff
@@ -356,3 +360,84 @@ def __init__(self, temporalmin: _u.s, temporalmax: _u.s):
 
     def collides(self, other):
         return isinstance(other, self.__class__)
+
+
+# highLevelSoftwareVersion
+class SoftwareVersion(_SimpleAttr):
+    """
+    Version of the software used to record the data at the summit.
+
+    Parameters
+    ----------
+    version : `str`
+        Version of the software to search for.
+    """
+    def __init__(self, version: str):
+        super().__init__(version)
+
+
+# workflowName
+class WorkflowName(_SimpleAttr):
+    """
+    Name of the calibrarion workflow used.
+
+    Parameters
+    ----------
+    workflow_name : `str`
+        Name of the workflow.
+    """
+    def __init__(self, workflow_name: str):
+        super().__init__(workflow_name)
+
+
+# workflowVersion
+class WorkflowVersion(_SimpleAttr):
+    """
+    Version of the calibration workflow used.
+
+    Parameters
+    ----------
+    workflow : `str`
+        Version of the workflow.
+    """
+    def __init__(self, workflow: str):
+        super().__init__(workflow)
+
+
+# observingProgramExecutionId
+class ObservingProgram(_SimpleAttr):
```

Comment:
I think add the `ExecutionID` parts to these, only real advanced users will care, so I think explicit is better.